roles/openshift_logging_fluentd/tasks/main.yaml

```diff
@@ -1,5 +1,8 @@
 ---
 - fail:
+    msg: The ES_COPY feature is no longer supported. Please remove the variable from your inventory
```

Comment:
If we're going to update the docs to state that the installer will fail then this is alright. Otherwise I think we should just print out a warning message that it will be ignored with `debug`

Reply:
Removed everything outright in our documentation: https://github.com/openshift/openshift-docs/pull/5319  Not sure what else you are suggesting here or what I need to do to modify

Reply:
I would say let's update documentation. And furthermore, is it possible to tell users to use secure_forward method instead?

Reply:
The documentation has a block that says 'forward to external aggregator' which is about secure forward.  Is it implied?

Reply:
I think if we keep that block and maybe append information how it will fail if `ES_COPY` is used then we should be covered.

Reply:
I added a block to the docs to explicitly state the feature is no longer supported and to use fluentd